src/components/JsxParser.js

```diff
@@ -45,7 +45,10 @@ export default class JsxParser extends React.Component {
       raw.replace(new RegExp(`(</?)${tag}`, 'ig'), '$1REMOVE')
     , rawJSX).trim()
 
-    const doc = parser.parseFromString(jsx, 'text/html')
+    const wrappedJsx = `<!DOCTYPE html>\n<html><body>${jsx}</body></html>`
```

Comment:
That's a good point, switching to `application/xhtml+xml` is was needed. I now added a check for it and also tests for the major use cases.